server/src/main/java/se2/ticktackbumm/server/network/ServerMessageHandler.java

```diff
@@ -87,4 +88,9 @@ public void handleBombStart(){
         int timer = new SecureRandom().nextInt(30)+1;
         serverMessageSender.sendBombStart(timer);
     }
+
+
+    public void handleSpinWheelFinished(GameMode gameMode) {
```

Comment:
Besser wäre es hier, die `GameData` in `ServerData` zu aktualisieren und dann eine `GameUpdate` Message an die Clients zu schicken. Sonst zerstört das nächste `GameUpdate` den aktuellen game mode.